backend/src/api/app_factory.py

```
@@ -139,14 +139,18 @@ def _configure_routes(app: FastAPI) -> None:
     """Configure application routes"""
 
     # Import route modules
-    from .routes import upload, status, enhanced_guide, preferences, generation, test_guide, test_quality
+    from .routes import upload, status, enhanced_guide, preferences, generation, pdf, pdf_html, health, debug, test_guide, test_quality
```

Ruff (0.12.2)

Redefinition of unused `upload` from line 13

Remove definition: `upload`

(F811)